test/project_types/script/commands/create_test.rb

```diff
@@ -38,6 +38,13 @@ def test_can_create_new_script
         perform_command
       end
 
+      def test_help
+        Script::Layers::Application::ExtensionPoints.expects(:types).returns(%w(ep1 ep2))
+        ShopifyCli::Context.expects(:message).with('script.create.help', ShopifyCli::TOOL_NAME,
+           '{{cyan:ep1}}, {{cyan:ep2}}')
```

Comment:
Nitpick: Same as my above comment. Another thing we can do instead is put chained method calls on different lines. Whichever you prefer!

eg. 
```ruby
ShopifyCli::Context
  .expects(:message)
  .with('script.create.help', ShopifyCli::TOOL_NAME,' {{cyan:ep1}}, {{cyan:ep2}}')
```